store/copr/region_cache.go

```diff
@@ -115,10 +118,11 @@ func (c *RegionCache) OnSendFailForBatchRegions(bo *Backoffer, store *tikv.Store
 		logutil.Logger(bo.GetCtx()).Info("Should not reach here, OnSendFailForBatchRegions only support TiFlash")
 		return
 	}
-	for _, ri := range regionInfos {
+	logutil.Logger(bo.GetCtx()).Info("Send fail for " + strconv.Itoa(len(regionInfos)) + " regions, will switch region peer for these regions.")
```

Comment:
update the log to these?
... will switch region peer for these regions, the first max(10,len(reginoInfos)) are ...?